src/wallet/rpcwallet.cpp

```diff
@@ -1129,19 +1129,16 @@ struct tallyitem
 UniValue ListReceived(const UniValue& params, bool fByAccounts)
 {
     // Minimum confirmations
-    int nMinDepth = 1;
-    if (params.size() > 0)
-        nMinDepth = params[0].get_int();
+    const int nMinDepth = params.size() == 0 ? 1 : params[0].get_int();
```

Comment:
Probably better to leave these alone. If params[0] is null, we really should silently use the default value... The longer version is also more readable/obvious.